model/metrics/system-metrics.yaml

```
@@ -20,8 +21,9 @@ groups:
     instrument: gauge
     unit: "1"
     attributes:
-      - ref: system.cpu.state
-        brief: "The CPU state for this data point. A system's CPU SHOULD be characterized *either* by data points with no `state` labels, *or only* data points with `state` labels."
+      - ref: cpu.mode
+        brief: "The CPU state for this data point. A system's CPU SHOULD be characterized *either* by data points with no `mode` labels, *or only* data points with `mode` labels."
```

```suggestion
        brief: "The CPU mode for this data point. A system's CPU SHOULD be characterized *either* by data points with no `mode` labels, *or only* data points with `mode` labels."
```

BTW you can avoid repetition by defining a new yaml attribute_group and extending it in metrics. E.g.

```yaml
  - id: metric.cpu.attributes
    type: attribute_group
    brief: Metrics attribute with CPU 
    attributes:
      - ref: cpu.mode
        brief: "The CPU mode for this data point. A system's CPU SHOULD be characterized *either* by data points with no `mode` labels, *or only* data points with `mode` labels."
        note: "Following states SHOULD be used: `user`, `system`, `nice`, `idle`, `iowait`, `interrupt`, `steal`"
      - ref: system.cpu.logical_number 
    
- id: metric.system.cpu.utilization
  ...
  extends: metric.cpu.attributes

- id: metric.system.cpu.time
  ...
  extends: metric.cpu.attributes
```